GeneralsMD/Code/Main/WinMain.cpp

```diff
@@ -373,6 +373,10 @@ LRESULT CALLBACK WndProc( HWND hWnd, UINT message,
                 case SC_SIZE:
                 case SC_MAXIMIZE:
                 case SC_KEYMENU:
+					// TheSuperHackers @bugfix Mauller 10/05/2025 Handle left ALT when in windowed mode to prevent game freezing
+					if( TRUE == ApplicationIsWindowed )
```

Comment:
Previously all of these returned 1 in fullscreen. What is the reason for this not doing that anymore?